tutorial/creating_packages/handle_sources_in_packages.rst

```diff
@@ -61,6 +61,8 @@ Let's have a look at the changes in the *conanfile.py*:
         default_options = {"shared": False, "fPIC": True}
 
         def source(self):
+            # Please, be aware that using the head of the branch instead of an immutable tag
+            # or commit is not a bad practice and not allowed by Conan
```

Comment:
I'd personally go for "strongly discouraged, unsupported by Conan and likely to cause issues"